plugins/header_rewrite/parser.h

```diff
@@ -69,6 +69,25 @@ class Parser
     return _val;
   }
 
+  void
+  set_regex(std::string cond, std::string pat)
+  {
+    _regex_cond = cond;
+    _regex_pat  = pat;
+  }
+
+  std::string &
+  get_regex_cond()
```

Comment:
These may make sense to declare as const, e.g.

```
std::string &
get_regex_cond() const {
```

assuming that we're not going to modify the string returned.